openshift-tmp/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
```

Comment:
Call the directory openshift instead of openshift-tmp, I would suggest. It is likely here to stay.